src/main/java/lotto/model/LottoResults.java

```diff
@@ -0,0 +1,61 @@
+package lotto.model;
+
+import java.util.Objects;
+
+public class LottoResults {
+    private final int countThreeMatches;
+    private final int countFourMatches;
+    private final int countFiveMatches;
+    private final int countSixMatches;
+    public final static int PRIZE_THREE_MATCHES = 5000;
+    public final static int PRIZE_FOUR_MATCHES = 50000;
+    public final static int PRIZE_FIVE_MATCHES = 1500000;
+    public final static int PRIZE_SIX_MATCHES = 2000000000;
```

Comment:
```suggestion
    public final static int PRIZE_THREE_MATCHES = 5000;
    public final static int PRIZE_FOUR_MATCHES = 50000;
    public final static int PRIZE_FIVE_MATCHES = 1500000;
    public final static int PRIZE_SIX_MATCHES = 2000000000;
    private final int countThreeMatches;
    private final int countFourMatches;
    private final int countFiveMatches;
    private final int countSixMatches;
```
컨벤션 위반
앞의 문자열 덧셈 계산기 사칙 연산과 같이 서로 관련성이 있는 상수 값은 enum으로 구현해 보는 것은 어떨까?